ui-components/src/lists/PreferencesList.scss

```diff
@@ -1,6 +1,24 @@
+.preferences-list {
+  --background-color: var(--bloom-color-white);
```

Comment:
I'm not seeing this one or the `info-card-title-text-transform` used anywhere

Reply:
removed the --info-card-title-text-transform! good catch.

the --bloom-color-white comes from [colors.scss](https://github.com/bloom-housing/bloom/blob/main/ui-components/src/global/tokens/colors.scss)

Reply:
The --background-color variable is not being used in this file.

Reply:
🤦 ofc. fixed!